### ISSUE TITLE
rebuild the suspended tab

### ISSUE THREAD

**Issue by Golddouble** [piroor/suspendtab#64]
I suspend a tab. After 1 hour the website behind the suspeded tab has changed. (For example because it is a forum with new answers.). After 2 hours a click the suspended tab. Now the website appears.

But it is not the actual website, it is the content which I have suspended.

My expectation was that I get the actual changed website. Is there anything I can adjust to get the expected result?

Thank you.

**Comment by piroor** [comment]
I agree that the behavior (reload after restored) is useful for some cases, but it will increase traffic. I think we should determine conditions to reload intelligently. Do you have any idea?

**Comment by Golddouble** [comment]
I would prefer a manual (not intelligent) reload function. For example clicking ALT+left mouse button.
